### PR TITLE
feature/admin-ui localization

### PR DESCRIPTION
**Pull request by gallayl**
- [x] Created Localization service
 - [x] Created Localization context with provider
 - [x] Improved Monaco model initialization
 - [x] Extracted hardcoded string values to *default* localization
 - [x] Added hungarian localization *stub* for testing localization change

### CODE REVIEW

File: apps/sensenet/package.json [hunk 1]
```diff
@@ -64,7 +64,9 @@
     "react-router-dom": "^4.3.1",
     "source-map-loader": "^0.2.4",
     "style-loader": "^0.23.1",
+    "ts-config-webpack-plugin": "^1.3.1",
     "ts-jest": "^24.0.0",
+    "ts-node": "^8.0.2",
```
Comment (B3zo0) [inline apps/sensenet/package.json]: ts-node is only necessary if the webpack config is a typescript file.

Comment (gallayl) [reply apps/sensenet/package.json]: 👍

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 2]
```diff
@@ -139,7 +151,7 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
                   <Tooltip
                     title={
                       <React.Fragment>
-                        {'Personal settings'} <br /> {'Customize the application behavior'}
+                        {localization.personalSettingsTitle} <br /> {localization.personalSettingsSecondaryText}
                       </React.Fragment>
                     }
                     placement="right">
```
[the remark this answers is not in the document]
Comment (gallayl) [reply apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: 👍

File: apps/sensenet/src/context/LocalizationContext.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  values: DefaultLocalization,
+})
+
+/**
+ * Context provider for Localization values. Update the PersonalSettings.language to load a new language into the context.
+ * @param props
+ */
+export const LocalizationProvider: React.FunctionComponent = props => {
+  const [localizationService] = useState(new LocalizationService())
+  const [currentLanguage, setCurrentLanguage] = useState('default')
```
[the remark this answers is not in the document]
Comment (gallayl) [reply apps/sensenet/src/context/LocalizationContext.tsx]: good catch, fixed in a recent commit

**Pull request merged**